ecs.php

```diff
@@ -39,6 +40,8 @@
         PropertyTypeHintSniff::class => null,
         PhpUnitStrictFixer::class => ['tests/ORM/TimestampableTest.php'],
         UnusedPrivateElementsSniff::class => ['tests/Fixtures/Entity/SluggableWithoutRegenerateEntity.php'],
+        FinalClassFixer::class => ['tests/Fixtures/Entity/TimestampableMappedSuperclassEntity.php'],
+        OrderedImportsFixer::class => ['tests/Fixtures/Entity/TimestampableMappedSuperclassEntity.php'],
```

Comment:
Instead of ignoring the error, let cs fix it.

```bash
composer fix-cs
```

Reply:
ecs suggests to remove `use DateTimeInterface` but it uses in `@var DateTimeInterface`, after that phpstan will be unhappy.

Reply:
I see. There is bug in one them, I'll look into that later, it's unrelated to this PR